From: Roger Light <roger@atchoo.org>
Date: Mon, 10 Mar 2025 22:52:17 +0100
Subject: Fix missing upstream test.

Forwarded: not-needed
---
 test/broker/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/broker/Makefile b/test/broker/Makefile
index c935247..e1501b4 100644
--- a/test/broker/Makefile
+++ b/test/broker/Makefile
@@ -135,7 +135,7 @@ test : test-compile 01 02 03 04 05 06 07 08 09 10 11 12 13 14
 	./06-bridge-reconnect-local-out.py
 
 07 :
-	./07-will-delay-invalid-573191.py
+	#./07-will-delay-invalid-573191.py
 	./07-will-delay-reconnect.py
 	./07-will-delay-recover.py
 	./07-will-delay-session-expiry.py
